flexget/manager.py

```diff
@@ -243,6 +245,9 @@ def execute(self, options=None, output=None, loglevel=None, priority=1):
             options_namespace.__dict__.update(options)
             options = options_namespace
         task_names = self.tasks
+        # Only reload config if daemon
+        if self.is_daemon and self.config_file_hash != self.hash_config():
```

Comment:
Once the config is loaded during init, the config_file_hash will be set in manager.